scripts/apps/authoring-react/authoring-integration-wrapper.tsx

```diff
@@ -378,6 +382,90 @@ export class AuthoringIntegrationWrapper extends React.PureComponent<IPropsWrapp
             <WithInteractiveArticleActionsPanel location="authoring">
                 {(panelState, panelActions) => (
                     <AuthoringReact
+                        extraActions={(() => {
+                            const actions = [];
+                            const {state} = this;
+
+                            if (appConfig.features.useTansaProofing !== true) {
+                                if (state.spellcheckerEnabled) {
+                                    const nextValue = false;
+
+                                    actions.push({
+                                        label: gettext('Disable spellchecker'),
+                                        onTrigger: () => {
```

Comment:
Spellchecking implementation needs to be either inside our outside authoring react. We won't be easily able to move it out, not sure even if it would make sense. Thus let's keep it inside. Then do not expose **implementation details** outside authoring react. If you need to interact with it - use `exposed` to expose the interface like

```
spellchecker: {
    enabled: boolean;
    enable(): void;
    disable(): void;
},
```